packages/account/src/wallet/wallets.ts

```diff
@@ -16,8 +16,8 @@ import { decryptKeystoreWallet } from './keystore-wallet';
 
 export interface GenerateOptions {
   /** Additional entropy for the random bytes */
-  entropy?: BytesLike;
-  provider?: Provider;
+entropy ?: BytesLike;
```

Comment:
Didn't know this was even possible, but I like the `entropy?:` without whitespace version

Reply:
Agreed.